tests/rules/test_remove_shell_prompt_literal.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from thefuck.rules.remove_shell_prompt_literal import match, get_new_command
+from thefuck.types import Command
+
+
+@pytest.mark.parametrize(
+    "command",
```

Comment:
How about parametrizing only `script` (as string) and declare a `Command` instance with `"$: command not found"` as `output`?

```suggestion
    "script",
```

That would make all test tables more readable.